ble/UUID.h

```diff
@@ -42,7 +97,7 @@ class UUID {
      *         different service or characteristics on the BLE device.
      *
      * @param longUUID
-     *          The 128-bit (16-byte) UUID value, MSB first (big-endian).
+     *          The 128-bit (16-byte) UUID value, LSB first (little-endian).
```

Comment:
I know we discussed this; but I'm very hesitant to commit to this switch. If we go ahead with this, it might require a bump to the major version.
Can you re-state what the problem is with the current API? Why can't we continue to accept 128-bit UUIDs in MSB form in the constructor? This will potentially break user code.